### PR TITLE
separate sponsor report and sub-report views into pipeline I/II

### PR DESCRIPTION
**Pull request by Jana-A**
populating strategies_id in project table + separating sponsor report and sub report live into pipelines I/II

### CODE REVIEW

**Comment by Jana-A** [comment]
Note that in order to obtain this view in the cached reports, cached reports need to only be regenerated.

**Comment by Jana-A** [comment]
@dparrysmith PR ready for review.

File: ddl/versions/132/fixtures.sql [hunk 1]
```diff
@@ -0,0 +1,13 @@
+UPDATE projects SET strategy_id = 'Pipeline I'
```
Comment (dparrysmith) [inline ddl/versions/132/fixtures.sql]: Looks fine. I think Mark may have updated the Decipher ones manually but the Pipeline I projects certainly need doing.

Maybe need to do a test to see if the strategy_id is NULL before setting ity so as not to overwrite anything that has been set manually? I think he did it directly in the database.
